book/src/Setup.md

```diff
@@ -71,7 +71,7 @@ If you want to use the `CSIDriver` CRD and get a preview of how configuration wi
 1) Ensure the feature gate is enabled with `--feature-gates=CSIDriverRegistry=true`
 2) Install the `CSIDriver` CRD on the Kubernetes cluster with the following command:
 ```
-$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/testdata/csidriver.yaml
+$> kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/testdata/csidriver.yaml --validate=false
```

Comment:
Thanks for adding this. Missed it.

Reply:
We'll should archive all of this after 1.13 as a 1.12 steps and update it for 1.13.

Reply:
Yep after 1.13 components roll out we could do another doc pass to make sure everything is updated